### ISSUE TITLE
Model AABB information in runtime

### ISSUE THREAD

**Issue by aglitchman** [defold/defold#9886]
**Is your feature request related to a problem? Please describe (REQUIRED):**
In a 3d game, I needed to dynamically fit objects that are in the scene into the camera view. For this purpose it is necessary to know the dimensions of the meshes of these objects, i.e. axis-aligned bounding boxes.

**Describe the solution you'd like (REQUIRED):**
It would be great if Model API would give access to this data about model meshes, because the engine uses it for frustum culling.

**Describe alternatives you've considered (REQUIRED):**
We have to store the path to models and their AABBs in separate files. And make additional loaders for such data. And this generally increases the size of the game assets. So you get such a duplication of the same data.

**Comment by aglitchman** [comment]
Related:
* https://github.com/defold/defold/issues/7626

**Comment by selimanac** [comment]
Yes, please :) In my case, I had set bounds at my native extension to calculate the aabb collisions. Currently I'm setting them manually, it would be great to get it from models(also using native  sdk)

**Comment by selimanac** [comment]
Just waiting on this—need it for my new extension :)
https://forum.defold.com/t/daabbcc-dynamic-tree-aka-aabb-tree-native-extension-for-fast-aabb-collision-detection/18328/98?u=selimanac
